sdk/storage/Azure.Storage.Blobs/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 12.5.0-preview.1 (Unreleased)
-
+## 12.4.2 (2020-05)
+- Fixed bug where blob, file and directory names were not URL encoded.
+- Fixed bug where BlobBaseClient.DownloadAsync() could corrupt downloaded data if intermittent network failure occurs.
```

Comment:
Might reword that as `download incorrectly` to avoid any confusion with data being corrupted on the service from a bad request.

Reply:
sounds good.